### PR TITLE
Bump versions of dependencies and Node.js

### PR DESCRIPTION
**Pull request by kola-er**
Add support for Node.js 8.10.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 7.0.0
+
+### cli
+### schema
+### core
+
+* (**BREAKING**) Apps with dependency zapier-platform-core=">=7.0.0" run only on Node.js `v8.10.0` in AWS Lambda. If you need to continue running on Node.js `v6.10.3`, use zapier-platform-core="<=6.1.0".
```
Comment (xavdid) [inline CHANGELOG.md]: this is generated automatically later, can revert.

File: README-source.md [hunk 2]
```diff
@@ -1179,7 +1179,7 @@ This makes it pretty straightforward to integrate into your testing interface. I
 ```yaml
 language: node_js
 node_js:
-  - "6.10.2"
+  - "8.10.0"
```
Comment (xavdid) [inline README-source.md]: this should be `LAMBDA_VERSION` so it gets autofilled.

File: README-source.md [hunk 3]
```diff
@@ -1179,7 +1179,7 @@ This makes it pretty straightforward to integrate into your testing interface. I
 ```yaml
 language: node_js
 node_js:
-  - "6.10.2"
+  - `LAMBDA_VERSION`
```
Comment (kola-er) [inline README-source.md]: This way? @xavdid

Comment (xavdid) [reply README-source.md]: oh sorry, I totally spaced on this. it would be "LAMBDA_VERSION" since I believe it's just a string replace.

**Comment by kola-er** [comment]
@xavdid Please final review. Thanks.

**Pull request merged**